plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/tls/CassandraSslConfig.java

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
> CassandraSslConfig.java

Did you forget renaming the file? Same for `TestCassandraSslConfig.java`.

Reply:
Oops it was missed. Now renamed the file